nemo/collections/nlp/models/language_modeling/megatron/bert/bert_model.py

```diff
@@ -154,6 +174,260 @@ def post_language_model_processing(
         return lm_loss, binary_logits
 
 
+from megatron.core.transformer.transformer_layer import TransformerLayerSubmodules
+
+
+@dataclass
+class TransformerLayerSubmodulesPostLNSupport(TransformerLayerSubmodules):
+    def __init__(self, post_att_layernorm, post_mlp_layernorm, **kwargs):
+        super(TransformerLayerSubmodulesPostLNSupport, self).__init__(**kwargs)
+        self.post_att_layernorm = post_att_layernorm
+        self.post_mlp_layernorm = post_mlp_layernorm
+
+
+class TransformerLayerPostLNSupport(TransformerLayer):
```

Comment:
Sorry for being nit picky, but could you also rename this to TransformerLayerWithPostLNSupport (Just like with block support). Same abovec for TransformerLayerSubmodulesWithPostLNSupport

Reply:
sure, I think I mistakenly stashed my changes yesterday, added these beck